src/JitInterface/src/CorInfoTypes.VarInfo.cs

```diff
@@ -26,10 +26,10 @@ public struct NativeVarInfo
     [StructLayout(LayoutKind.Sequential)]
     public struct VarLoc
     {
-        IntPtr A; // vlType + padding
-        int B;
-        int C;
-        int D;
+        public IntPtr A; // vlType + padding
```

Comment:
Maybe we could make this struct more useful instead of placing `VarLocType` into a separate file and reinterpreting the struct there?

Reply:
I added an accessor for the location type so we're not re-interpreting A. The data int fields I'll leave as is. It would be overkill defining a set of overlapping explicit layout fields to try and make this type look like the C++ union equivalent given we use this thing in one place currently.